_data/nav.yml

```diff
@@ -1,3 +1,5 @@
+- title: Home
+  url: /
```

Comment:
I'm not sure if I agree with that from a usability point, but I'll remove this change.